test/integration/targets/docker_network/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create random name prefix
+  set_fact:
+    name_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+    cnames: []
+    dnetworks: []
+
+- debug:
+    msg: "Using name prefix {{ name_prefix }}"
+
+- block:
+  - include_tasks: run-test.yml
```

Comment:
use `- include_tasks: "{{ item }}"` directly, no need to use `run-test.yml` (`- include_tasks: tests/basic.yml` works too)

Reply:
There's one big advantage over doing it this way: if you do all `include_tasks` directly, you will get the `included: .../tasks/tests/xxx.yml for localhost` all at the same time, while with this indirect include, you will get them before the tasks are actually executed. This makes it easier to find out in which file a task is defined (if you're hunting down something that breaks).

Reply:
(I originally did it directly for the docker_container tests, but after I had two tests instead of only one, I noticed this and changed it this way to make it easier to find out to which "sub-test" tests belong to.)